test/e2e/join_and_leave_test.go

```diff
@@ -114,3 +121,51 @@ var _ = Describe("Test member cluster join and leave flow", Ordered, Serial, fun
 		ensureCRPAndRelatedResourcesDeletion(crpName, allMemberClusters)
 	})
 })
+
+var _ = Describe("Test member cluster force delete flow", Ordered, Serial, func() {
+	Context("Test cluster join and leave flow with member agent down and force delete member cluster", Ordered, Serial, func() {
+		It("Simulate the member agent going down in member cluster", func() {
+			Eventually(func() error {
+				return updateMemberAgentDeploymentReplicas(memberCluster3WestProdClient, 0)
+			}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to simulate member agent going down")
+		})
+
```

Comment:
it just occurred to me that we didn't verity that there are no more pods in the deployment